freemocap/fmc_runme.py

```diff
@@ -249,8 +251,11 @@ def RunMe(sessionID=None,
             if place_skeleton_on_origin:
                 sesh.mediaPipeSkel_fr_mar_xyz_smoothed_unrotated = sesh.mediaPipeSkel_fr_mar_xyz.copy()
                 np.save(sesh.dataArrayPath/'mediaPipeSkel_3d_smoothed_unrotated.npy', sesh.mediaPipeSkel_fr_mar_xyz_smoothed_unrotated) #save data to npy
+                good_frame = fmc_good_frame_finder.find_good_frame(sesh.mediaPipeSkel_fr_mar_xyz, .6, debug = False) #.6 is an initial guess for the velocity. seems to be a safe bet for for all the cases I've tried
```

Comment:
again, just pointing it out for posterity's sake - We're introducing a conflict with the term "good frame" here, as that concept is now used in multiple ways within our code (here for the origin alignment, but there's another "good frame" referenced in the Blender code.